src/glb-redirect/ipt_GLBREDIRECT.c

```diff
@@ -605,23 +663,84 @@ static struct xt_target glbredirect_tg4_reg __read_mostly = {
 	.me		    = THIS_MODULE,
 };
 
+static int proc_show(struct seq_file *m, void *v)
+{
+	unsigned int cpu, start;
+	struct glbgue_stats sum = {0};
+
+	for_each_possible_cpu(cpu) {
+		struct glbgue_stats *s = per_cpu_ptr(percpu_stats, cpu);
+
+		do {
+			start = u64_stats_fetch_begin(&s->syncp);
+			sum.total_packets += s->total_packets;
```

Comment:
I'm not sure the retry logic guarantees that these values are safe to use here yet - it looks like [typically](https://github.com/torvalds/linux/blob/master/drivers/net/loopback.c#L112-L118) other kernel sources retrieve the values to local variables each time through the loop, then use those final values once the retry loop completes.

Reply:
Good catch, I've pushed a fix.